students/luyao_xu/lesson04/mailroom2.py

```diff
@@ -0,0 +1,104 @@
+#!/Library/Frameworks/Python.framework/Versions/3.7/bin/python3
+don_1 = {'name': 'Amy Walker', 'donation amount': [18900.90, 4500]}
+don_2 = {'name': 'Peter Thomson', 'donation amount': [9999.99, 500, 6000.88]}
+don_3 = {'name': 'July Jensen', 'donation amount': [5300.00, 200]}
+don_4 = {'name': 'Paul Allen', 'donation amount': [320.57]}
+don_5 = {'name': 'Jenny Palmer', 'donation amount': [66.89]}
+donations = [don_1, don_2, don_3, don_4, don_5]
```

Comment:
Suggest using a dictionary as the main data structure, this will simplify your code by quite a bit. Example:
```
donations = {"Paul Allen": [200, 100], "Peter Thomson": [33443, 22]}
```